src/main/java/no/fintlabs/adapter/ResourcePublisher.java

```diff
@@ -0,0 +1,27 @@
+package no.fintlabs.adapter;
+
+import no.fint.model.resource.FintLinks;
+import no.fintlabs.adapter.models.AdapterCapability;
+
+import java.util.List;
+import java.util.concurrent.SubmissionPublisher;
+
+public abstract class ResourcePublisher<T extends FintLinks, R extends ResourceRepository<T>> extends SubmissionPublisher<List<T>> {
```

Comment:
We are considering moving this **adapter** package to a separate library? Given the open-closed principle, this code may not be open to modification.

Reply:
Yes, definitely, but I think we need to test it a bit more first.